### ISSUE TITLE
Product Query [3]: [Variation] `Best Selling Products` block

### ISSUE THREAD

**Issue by sunyatasattva** [woocommerce/woocommerce-gutenberg-products-block#7325]
Implement a “Best Selling Products” variation of the “Product Query” block. This variation should show at least the following settings:

* Category
* Stock status

This is blocked by #7218 and #7324

**Comment by sunyatasattva** [comment]
Closing as won't do as of this comment: pdnLyh-2jZ-p2#comment-1685
